### ISSUE TITLE
Slider Gap controller issues

### ISSUE THREAD

**Issue by bentalgad** [dudaster/ele-custom-skin#160]
When changing the "Slider Gap" value with the slider the number doesn't indicate the values:

![2020-03-09 11 39 46](https://user-images.githubusercontent.com/23374215/76200789-e0d51280-61fa-11ea-9a14-d418d8413df7.gif)

And other times it's just now working at all.

**Comment by dudaster** [comment]
I'll check it on my end... Does it work on other widgets? Because that's clearly an Elementor issue. Does it throw an JS error of somekind? 

What is the version of the elementor you're using?

**Comment by bentalgad** [comment]
I don't think so. i tried with 3 different versions of Elementor pro (2.8.5, 2.8.4, 2.8.3), and also when i use the regular "cards" layout it works perfectly

**Comment by bentalgad** [comment]
Also when using the custom layout with loop template it works. it stops working only when using the "Show in slider" option.

**Comment by dudaster** [comment]
What browser are you using ?

**Comment by bentalgad** [comment]
latest chrome, mac.

**Comment by dudaster** [comment]
I cannot reproduce the issue. But I see in your video that the things at your side start from right to left. Maybe can this be the difference I cannot reproduce it?

**Comment by bentalgad** [comment]
It's a RTL Wordpress Installation (Hebrew).
